docker-compose.yml

```diff
@@ -1,16 +1,27 @@
-postgres:
-  image: postgres:9.4.1
-box:
-  image: busybox
-  volumes:
-    - /box
-web:
-  build: .
-  ports:
-    - "3000:3000"
-  links:
-    - postgres
-  volumes:
-    - .:/app
-  volumes_from:
-    - box
+services:
+  postgres:
+    image: postgres:14.12
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: UNSECURE_LOCAL_DEVELOPMENT_PASSWORD
+
+  box:
```

Comment:
If we don't suggest to use box&web anymore, should we remove them then?

Reply:
total agree